packages/phishing-controller/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.2]
+### Fixed
+- Create a Set from `metamaskConfig.blocklist` in the `PhishingController`. Use a `Set` + `has` method instead of `includes` in array ([#1086](https://github.com/MetaMask/core/pull/1086))
```

Comment:
Needs a newline to pass lint I believe.

Also here's a suggested re-wording to use a high-level description first, with the more technical one as a sub-bullet:
```suggestion
- Improve performance of phishing list update ([#1086](https://github.com/MetaMask/core/pull/1086))
  - We now use a `Set` + `has` method instead of the array `includes` method for detecting overlap between phishing lists after an update.

```